treeherder/client/thclient/client.py

```diff
@@ -674,16 +674,22 @@ def get_machine_platforms(self):
         """
         return self._get_json(self.MACHINE_PLATFORM_ENDPOINT)
 
-    def get_resultsets(self, project, **params):
+    def get_pushes(self, project, **params):
         """
-        Gets resultsets from project, filtered by parameters
+        Gets pushes from project, filtered by parameters
 
-        By default this method will just return the latest 10 result sets (if they exist)
+        By default this method will just return the latest 10 pushes (if they exist)
 
         :param project: project (repository name) to query data for
         :param params: keyword arguments to filter results
         """
-        return self._get_json_list(self.RESULTSET_ENDPOINT, project, **params)
+        return self._get_json_list(self.PUSH_ENDPOINT, project, **params)
+
+    def get_resultsets(self, project, **params):
+        """
+        Returns get_pushes for backwards compatibility
+        """
+        return self.get_pushes(project)
```

Comment:
Ah by the "..." I meant an appropriate deprecation message, eg "DEPRECATED: TreeherderClient.get_resultsets(), use get_pushes() instead."